docs/static/includes/module-features/bicepFeatures.csv

```diff
@@ -7,7 +7,7 @@
 6,res/app/container-app,✅,✅,✅,,,,✅
 7,res/app/job,✅,✅,✅,,,,✅
 8,res/app/managed-environment,✅,✅,✅,,,,✅
-9,res/automation/automation-account,✅,✅,✅,✅,✅,✅,✅
+9,res/automation/automation-account,✅,✅,✅,,,✅,
```

Comment:
Blocking comment, this requires troubleshooting

Reply:
Si si. On it. Thanks @eriqua

Reply:
Ran locally & was not able to reproduce. I'll continue testing with the workflow, but so far there's no indication why it would want to commit the change